### DISCUSSION TITLE
how to customize the highlight colors that respond to the style of `tokyonight` theme?

### DISCUSSION THREAD

**Discussion by hopezh** [folke/flash.nvim#238] [Q&A]
As shown below, the highlight colors of `flash` are different btw the night and day styles of `tokyonight`. 

![2023-09-13-11-18-18](https://github.com/folke/flash.nvim/assets/23435810/97505554-d3cb-4c50-bcb1-3e8d22bbbbd4)

![2023-09-13-11-21-07](https://github.com/folke/flash.nvim/assets/23435810/13d6bd4c-99c0-4bb2-9c9c-0e6f301519ee)

May I ask:
1. How to customize the **highlight colors**, e.g. FlashBackdrop, FlashMatch, FlashLabel, etc.? 
2. How to make them respond to the "**styles**" of the `tokyonight` theme currently used? 

Thanks.

**Comment by nshern** [comment]
I would also very much like to know this. When using [Oxocarbon](https://github.com/nyoom-engineering/oxocarbon.nvim), the labels are indistinguishable from the rest of the text which makes the color scheme unusable with flash.

**Comment by iandol** [comment]
Can anyone please help for how to configure the highlights with flash? They are really difficult to see with the catppuccin theme. Thanks.
